framework/include/materials/MaterialPropertyInterface.h

```diff
@@ -186,7 +186,7 @@ class MaterialPropertyInterface
   Moose::MaterialDataType _material_data_type;
 
   /// Pointer to the material data class that stores properties
-  MooseSharedPointer<MaterialData> _material_data;
+  std::shared_ptr<MaterialData> _material_data;
```

Comment:
Cody suggested just switching MooseShared -> std::shared whenever I run into it reworking code.